src/providers/object-provider.js

```diff
@@ -31,6 +31,7 @@ import { OBJECT_TYPES, NAMESPACE } from '../const.js';
 import OperatorStatusParameter from './user/operator-status-parameter.js';
 import { createCommandsObject } from './commands.js';
 import { createEventsObject } from './events.js';
+import { getMissionRoleFromParameter, getPossibleMissionRoleStatusesFromParameter, isMissionStatusParameter } from './mission-status/mission-status-parameter.js';
```

Comment:
This pattern is better than the one used for OperatorStatusParameter, thanks. In fact, can we please convert OperatorStatusParameter to use the same pattern for consistency?

Reply:
🫡